lib/editor/components/timetable/TimetableEditor.js

```diff
@@ -18,7 +18,7 @@ import type {Feed, FetchStatus, Pattern, StopTime, TimetableColumn, Trip, TripCo
 import type {TripValidationIssues} from '../../selectors/timetable'
 import type {EditorTables, TimetableState} from '../../../types/reducers'
 import { mergePatternHaltsOfPattern } from '../../../gtfs/util'
-import { patternHaltIsStop } from '../../util/location'
+import { patternHaltIsLocation, patternHaltIsStop } from '../../util/location'
```

Comment:
After adding a location group in a route pattern, the UI doesn't seem to pick up changes to the cells for the location group, so it is not possible to edit the scheduled times at location groups.

Reply:
I've tried to but failed to replicate this. Are you able to provide more detailed steps to reproduce?